pwndbg/commands/cymbol.py

```diff
@@ -251,84 +261,91 @@ def show_custom_structure(custom_structure_name: str, custom_structure_path: str
 
 
 parser = argparse.ArgumentParser(
-    description="Add, show, load, edit, or delete custom structures in plain C."
-)
-parser.add_argument(
-    "-a",
-    "--add",
-    metavar="name",
-    help="Add a new custom structure",
-    default=None,
-    type=str,
-)
-parser.add_argument(
-    "-f",
-    "--file",
-    metavar="filepath",
-    help="Add a new custom structure from header file",
-    default=None,
-    type=str,
-)
-parser.add_argument(
-    "-r",
-    "--remove",
-    metavar="name",
-    help="Remove an existing custom structure",
-    default=None,
-    type=str,
-)
-parser.add_argument(
-    "-e",
-    "--edit",
-    metavar="name",
-    help="Edit an existing custom structure",
-    default=None,
-    type=str,
-)
-parser.add_argument(
-    "-l",
-    "--load",
-    metavar="name",
-    help="Load an existing custom structure",
-    default=None,
-    type=str,
+    description="Manage custom C structures in pwndbg. Supports project-specific auto-loading from .gdbinit."
 )
-parser.add_argument(
-    "-s",
-    "--show",
-    metavar="name",
-    help="Show the source code of an existing custom structure",
-    default=None,
-    type=str,
+
+subparsers = parser.add_subparsers(dest="subcommand", help="Available subcommands")
+
+add_parser = subparsers.add_parser("add", help="Add a custom structure")
+add_parser.add_argument("name", help="Name of custom structure")
+add_parser.add_argument(
+    "--force", action="store_true", help="Overwrite if structure already exists"
 )
 parser.add_argument(
     "--show-all",
     help="Show names of all available custom structures",
     action="store_true",
 )
 
+remove_parser = subparsers.add_parser("remove", help="Remove a custom structure")
+remove_parser.add_argument("name", help="Name of custom structure")
+
+edit_parser = subparsers.add_parser("edit", help="Edit a custom structure")
+edit_parser.add_argument("name", help="Name of custom structure")
+
+load_parser = subparsers.add_parser("load", help="Load a custom structure")
+load_parser.add_argument("name", help="Name of custom structure")
+
+show_parser = subparsers.add_parser("show", help="Show a custom structure")
+show_parser.add_argument("name", help="Name of custom structure")
+
+file_parser = subparsers.add_parser("file", help="Add a structure from a header file")
+file_parser.add_argument("path", help="Path to header file")
+file_parser.add_argument("--name", help="Optional structure name")
+file_parser.add_argument("--force", action="store_true", help="Overwrite if exists")
+
+show_all_parser = subparsers.add_parser("show-all", help="Show all stored structure")
+
 
-@pwndbg.commands.Command(parser, category=CommandCategory.MISC)
+@pwndbg.commands.Command(
+    parser,
+    category=CommandCategory.MISC,
+    notes="""
+
+The `cymbol` command loads custom C structs and symbols into GDB using GCC under the hood.
```

Comment:
```suggestion
The `cymbol` command loads custom C structs and symbols into the debugger using GCC under the hood.
```
While it is true that `cymbol` isn't available for LLDB, it may be in the future.